### ISSUE TITLE
API level

### ISSUE THREAD

**Issue by seece** [SDLash3D/xash3d-android-project#21]
In `project.properties` the build target is defined as `target=android-8`, but in [the Travis build](https://travis-ci.org/SDLash3D/xash3d-android-project#L850) the value is overridden to be `android-15`.

     sed -i -- s/android-8/android-15/g project.properties

Also, in `AndroidManifest.xml` target SDK version is set to 14.

```xml
<uses-sdk android:minSdkVersion="8" android:targetSdkVersion="14" />
```

Why not set the API level to 15 already in the repo, instead of changing it manually before build?

**Comment by a1batross** [comment]
It's a dirty hack for android build tools. If build it with old API, build will fail. If build it with >15, it works. 

But android project can work on 2.3. 

This should be fixed.

**Comment by a1batross** [comment]
Fixed in 5bda511
